website/docs/getting-started/setup/instance-configuration/email/README.md

```diff
@@ -7,76 +7,88 @@ sidebar_position: 2
 
-    * **SMTP Host** - SMTP host of your email service provider
-    * **SMTP Port** - SMTP port of your email service provider
-    * **From Address** - a verified email address to be shown in the <mark >`from field`</mark> when users receive an email
-    * **TLS Protected Connection** - Bey default is enabled. Toggle back to disable
-    * **SMTP Username** - Add the username for your email service provider
-    * **SMTP Password** - Add the password for your email service provider
+5. Click the **SEND TEST EMAIL** button to verify the configuration. A toast message appears at the top of the page, indicating the success or failure of the test. Additionally, a test email is sent to your inbox on successful verification.
 
+6. Click the **SAVE & RESTART** button to save the configurations and restart the instance with the updated settings.
 
-* Click the <mark >`SAVE & RESTART`</mark> button to save the configurations and restart the Appsmith instance with the updated settings.
+### Environment variables
 
-:::tip
-Once your instance is restarted, you can use the **`SEND TEST EMAIL`** button to send a test email. You should see a message at the top of the page telling you whether the test succeeded or failed. On success, you should also receive a test email in your email inbox.
-:::
+Follow these steps to configure your email provider using environment variables:
+
+1. Go to your Appsmith instance configuration file. For example, the `docker-compose.yml` file for Docker and the `values.yaml` file for Kubernetes.
+
+2. Update the values of the environment variables as shown below:
+
+   |Variable	| Description |
+   |-----------|--------------|
+   | `APPSMITH_MAIL_ENABLED` | Set it to true to enable the email service. |
+   | `APPSMITH_MAIL_FROM`	| Set it to the verified email of the sender. |
+   | `APPSMITH_REPLY_TO`	| Set it to the email that should receive replies by default. |
+   | `APPSMITH_MAIL_HOST`	| Set it to the SMTP Host of the email service provider. |
+   | `APPSMITH_MAIL_PORT`	| Set it to the SMTP Port of the email service provider. |
+   | `APPSMITH_MAIL_SMTP_TLS_ENABLED` |	Set it to `true to enable transport layer security. |
+   | `APPSMITH_MAIL_SMTP_AUTH`	| Set it to share the credentials `APPSMITH_MAIL_USERNAME` and `APPSMITH_MAIL_PASSWORD` with the SMTP server. |
+   | `APPSMITH_MAIL_USERNAME` |	Set it to the username for accessing the SMTP service provider.
+   | `APPSMITH_MAIL_PASSWORD` |	Set it to the password for the SMTP user. Alternatively, you can use the API key generated by the email service provider for the SMTP user. |
 
-## Configuration guides
+3. Restart the instance.
 
-Follow the below guides to configure popular email service providers:
+## Troubleshooting
 
+If you have received a test email but can't send invitations then verify your email configuration again. If you continue to face issues, contact the support team using the chat widget at the bottom right of this page.
```

Comment:
we can also add this link - https://docs.appsmith.com/help-and-support/troubleshooting-guide/deployment-errors#unable-to-send-emails

Reply:
I searched for it, but didn't expect this to be under deployment errors :)